Source/SwiftLintFramework/Rules/NumberSeparatorRule.swift

```diff
@@ -113,10 +113,18 @@ public struct NumberSeparatorRule: OptInRule, CorrectableRule, ConfigurationProv
         return prefixes.filter { lowercased.hasPrefix($0) }.isEmpty
     }
 
-    private func isValid(number: String, reversed: Bool) -> (Bool, String) {
+    private func isValid(number: String, reversed: Bool, isFraction: Bool) -> (Bool, String) {
```

Comment:
you can remove the `reversed` parameter now, since it's always equivalent to `!isFraction`